pkg/plugin/trivy/plugin.go

```diff
@@ -145,8 +145,15 @@ func (c Config) GetServerInsecure() bool {
 }
 
 func (c Config) GetDBRepositoryInsecure() bool {
-	_, ok := c.Data[keyTrivyDBRepositoryInsecure]
-	return ok
+	val, ok := c.Data[keyTrivyDBRepositoryInsecure]
+	if !ok {
+		return false
+	}
+	boolVal, err := strconv.ParseBool(val)
```

Comment:
we could remove the `if` if we prefer, cos `boolVal` will be false in case of an error.
```suggestion
	boolVal, _ := strconv.ParseBool(val)
	return boolVal
```

Reply:
ok , I hope it will not fail lint